sprocs/assessment_instances_insert.sql

```diff
@@ -26,19 +28,61 @@ BEGIN
 
     -- ######################################################################
     -- determine the "number" of the new assessment instance
+    IF group_work THEN
+        SELECT gu.group_id
+        INTO  groupId
+        FROM  group_users as gu
+        JOIN  groups AS g ON(g.id = gu.group_id)
```

Comment:
Formatting: use the same spacing conventions as surrounding code. For example, here there should be a space after `ON`. This comment applies everywhere in this PR, not just here.